app/containers/ITunesSearch/index.js

```diff
@@ -0,0 +1,132 @@
+/**
+ *
+ * ITunesSearch
+ *
+ */
+
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { createStructuredSelector } from 'reselect';
+import debounce from 'lodash/debounce';
+import get from 'lodash/get';
+import isEmpty from 'lodash/isEmpty';
+import { Input, Skeleton } from 'antd';
+import styled from 'styled-components';
+import { compose } from 'redux';
+import { useInjectSaga } from '@utils/injectSaga';
+import { makeSelectITunesSearch, selectArtistName, selectTracksData, selectTracksError } from './selectors';
+import { iTunesSearchCreators } from './reducer';
+import saga from './saga';
+
+const { Search } = Input;
+
+const Container = styled.div`
+  && {
+    display: flex;
+    flex-direction: column;
+    max-width: ${props => props.maxWidth}px;
+    width: 100%;
+    margin: 0 auto;
+  }
+`;
+export function ITunesSearch({
+  maxWidth,
+  dispatchFetchTracks,
+  dispatchClearTracks,
+  artistName,
+  tracks = {},
```

Comment:
this is an issue in the react-template as well, use the default props instead of assigning a default value here. 

if you can create a PR fixing it in the react-template too that would be awesome!